Blueprints/BlueprintDefinitions/Msbuild-NETCore_2_1/CustomRuntimeFunction-FSharp/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -0,0 +1,43 @@
+# AWS Lambda Custom Runtime Function Project
+
+This starter project consists of:
+* Function.fs - contains a Main method function that starts the bootstrap, and a single function handler
+* aws-lambda-tools-defaults.json - default argument settings for use with Visual Studio and command line deployment tools for AWS
+* bootstrap - a Linux bash script that is invoked by the AWS Lambda infrastructure to start the function
+
+You may also have a test project depending on the options selected.
+
+The generated Main method is the entry point for the function's process.  The main method wraps the function handler in a wrapper that the bootstrap can work with.  Then it instantiates the bootstrap and sets it up to call the function handler each time the AWS Lambda function is invoked.  After the set up the bootstrap is started.
+
+The generated function handler is a simple method accepting a string argument that returns the uppercase equivalent of the input string. Replace the body of this method, and parameters, to suit your needs. 
+
+## Here are some steps to follow from Visual Studio:
+
+(Deploying and invoking custom runtime functions is not yet available in Visual Studio)
+
+## Here are some steps to follow to get started from the command line:
+
+Once you have edited your template and code you can deploy your application using the [Amazon.Lambda.Tools Global Tool](https://github.com/aws/aws-extensions-for-dotnet-cli#aws-lambda-amazonlambdatools) from the command line.  Version 3.1.3
```

Comment:
Change to 3.1.4 since we're going to do another release of the tools.
Do you mind changing the C# sample too while you're at it?

Reply:
Fixed and updated C# sample